src/Illuminate/Http/Concerns/InteractsWithFlashData.php

```diff
@@ -2,17 +2,24 @@
 
 namespace Illuminate\Http\Concerns;
 
+use Illuminate\Support\Arr;
+use Illuminate\Database\Eloquent\Model;
```

Comment:
Hey @driesvints 

Out of interest, why do you feel that should be avoided? Not asking because I think you are wrong, just interested, so I can learn the thinking, so I can consider it when I make PRs.

Is there an alternate approach you would take in this instance? Maybe checking if the class being passed in has an `getAttributes()` and `getAttribute()` method instead? Or an agnostic interface?

Reply:
People can use the Illuminate components separately. If they pull in this component, use this concern, they'll run into a fatal error because the database component isn't pulled in with it.

Reply:
Ah yes, that makes perfect sense! I will be sure to look out for that in the future. I have pushed some new commits changing the behaviour to decouple it from the database, and instead it just checks to see if it's an object and the object has a `getAttribute` method. If it does, it will use the value returned from that method, if not, will default to the value passed in as default.